client/app/components/saveable-form.js

```diff
@@ -39,26 +38,15 @@ export default class SaveableFormComponent extends Component {
     this.submittableChanges.validate();
   }
 
-  @action
-  async save() {
-    await this.saveChangeset.perform(this.args.onSave, this.saveableChanges);
-  }
-
-  @action
-  async submit() {
-    await this.saveChangeset.perform(this.args.onSubmit, this.submittableChanges);
-  }
-
-  @task(function* (callback, changeset) {
+  async saveChangeset(changeset, callback) {
```

Comment:
Looks like we pass in `this.saveableChanges` to the callback parameter. Can you explain why the parameter is called callback?

Reply:
dashingd3js.com/lessons/javascript-callback-functions#:~:text=A%20JavaScript%20Callback%20Function%20is,be%20used%20synchronously%20or%20asynchronously

Reply:
https://en.wikipedia.org/wiki/Callback_(computer_programming)

Reply:
should it be some other name?